README.md

```diff
@@ -1,14 +1,20 @@
-# solid-namespace
+# Solid Namespace
 [![](https://img.shields.io/badge/project-Solid-7C4DFF.svg?style=flat)](https://github.com/solid/solid)
 [![NPM Version](https://img.shields.io/npm/v/solid-namespace.svg?style=flat)](https://npm.im/solid-namespace)
 
 A collection of common RDF namespaces used in the Solid project.
 
+solid-namespace is meant to be used with [rdflib.js](/linkeddata/rdflib.js). It exports a collection of `NamedNode` values for commonly used namespaces, thereby shortcutting the need to call `$rdf.namespace(...)`.
```

Comment:
Meant to be used with any [RDF/JS](http://rdf.js.org/)-compatible library, including rdflib.js.